patterns/base/select/select.twig

```diff
@@ -0,0 +1,6 @@
+<select name="{{select.name}}" id="{{select.id}}" class="{{select.class}}">
+  {% for i in range(1, 3) %}
```

Comment:
don't use this loop here. it should be getting an object of `options`, which it will cycle through:

```
{{ for option in options }}
```